diff --git a/arch/arm64/boot/dts/qcom/sm8150.dtsi b/arch/arm64/boot/dts/qcom/sm8150.dtsi
index 4c9b7355179c..481bf384843f 100644
--- a/arch/arm64/boot/dts/qcom/sm8150.dtsi
+++ b/arch/arm64/boot/dts/qcom/sm8150.dtsi
@@ -2108,10 +2108,10 @@ ufs_mem_hc: ufshc@1d84000 {
 				<&gcc GCC_UFS_PHY_RX_SYMBOL_0_CLK>,
 				<&gcc GCC_UFS_PHY_RX_SYMBOL_1_CLK>;
 			freq-table-hz =
-				<37500000 300000000>,
+				<37500000 201500000>,
 				<0 0>,
 				<0 0>,
-				<37500000 300000000>,
+				<37500000 201500000>,
 				<0 0>,
 				<0 0>,
 				<0 0>,
